protoc_plugin/lib/file_generator.dart

```diff
@@ -292,7 +292,7 @@ class FileGenerator extends ProtobufContainer {
 
     out.println('// ignore: UNUSED_SHOWN_NAME');
     out.println(
-        '$_coreImport show int, bool, double, String, List, Map, override;\n');
+        '$_coreImport show int, bool, double, String, List, Map, override, Deprecated;\n');
```

Comment:
Now you're here, could you sort this show-list?

Reply:
Done.